src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -98,6 +98,9 @@
     <_AggressiveAttributeTrimming Condition="'$(_AggressiveAttributeTrimming)' == ''">true</_AggressiveAttributeTrimming>		
     <NullabilityInfoContextSupport Condition="'$(NullabilityInfoContextSupport)' == ''">false</NullabilityInfoContextSupport>
     <BuiltInComInteropSupport Condition="'$(BuiltInComInteropSupport)' == ''">false</BuiltInComInteropSupport>    
+    <!-- Verify DI trimmability at development-time, but turn the validation off for Release builds. -->
+		<VerifyDependencyInjectionOpenGenericServiceTrimmability Condition="'$(VerifyDependencyInjectionOpenGenericServiceTrimmability)' == '' and '$(Configuration)' == 'Release'">false</VerifyDependencyInjectionOpenGenericServiceTrimmability>
```

Comment:
Should this maybe check if `$(PublishTrimmed)` is true instead of `$(Configuration)`?

A lot of times developer might make a new configuration named `AppStore` or something, and they would configure it like a `Release` build.

Reply:
Yeah, that's probably a better property to switch on. Unless there is a better property.